config/environments/production.rb

```diff
@@ -55,7 +55,7 @@
   # config.cache_store = :mem_cache_store
 
   # Enable serving of images, stylesheets, and JavaScripts from an asset server.
-  # config.action_controller.asset_host = "http://assets.example.com"
+  config.action_controller.asset_host = "http://comfy.moneyadviceservice.org.uk"
```

Comment:
This config is served by production, but dev/test is diff. If something that could differ from environment it is better to move to a configuration management or an ENV var You already have https://github.com/moneyadviceservice/cms/blob/master/.env.dev where you can use those variables.